### PR TITLE
[KEYCLOAK-16455][Adapter - JavaScript] Propagate 3rd party cookies check

### PR DESCRIPTION
**Pull request by mhajas**
errors outside of JS adapter

Slightly updated PR: https://github.com/keycloak/keycloak/pull/7901 provided by @afedotov

### CODE REVIEW

File: adapters/oidc/js/src/main/resources/keycloak.d.ts [hunk 1]
```diff
@@ -178,6 +178,12 @@ declare namespace Keycloak {
 		 * @default false
 		 */
 		enableLogging?: boolean
+
+		/**
+		 * Configures how log will Keycloak adapter wait for receiving messages from server in ms
```
Comment (acontreras89) [inline adapters/oidc/js/src/main/resources/keycloak.d.ts:183]: > ... how lo**n**g ...

Comment (mhajas) [reply adapters/oidc/js/src/main/resources/keycloak.d.ts]: fixed. Thanks!

**Comment by mhajas** [comment]
I updated the PR based on the discussion here: https://github.com/keycloak/keycloak/pull/7901#issuecomment-861683735

**Review by jonkoops — APPROVED** [review]
This change is looking great!

**Review by stianst — APPROVED** [review]
LGTM

**Pull request merged**